_targets/build.project.armv7a7-imx6ull

```diff
@@ -34,7 +34,7 @@ export PORTS_CURL=y
 #
 SIZE_PAGE=$((0x1000))
 PAGE_MASK=$((~(SIZE_PAGE - 1)))
-KERNEL_OFFS=$((0x0))          # Disk image kernel offset (in bytes)
+KERNEL_OFFS=$((0x0))              # Disk image kernel offset (in bytes)
```

Comment:
unneeded change?

Reply:
I wanted to align this comment with simmilar comments in section below